docs/samples/README.md

```diff
@@ -1,61 +1,99 @@
-## KFServing Examples
-
-### Deploy KFServing InferenceService with out of the box Predictor
-[SKLearn Model](./sklearn)
-
-[PyTorch Model](./pytorch)
-
-[Tensorflow Model](./tensorflow)
-
-[XGBoost Model](./xgboost)
-
-[ONNX Model with ONNX Runtime](./onnx)
-
-[Simple String Model with NVIDIA Triton Inference Server](./triton/simple_string)
-
-[Serve BERT Model with NVIDIA Triton Inference Server](./triton/bert)
-
-### Deploy KFServing InferenceService with a custom Predictor
-
-[Hello World Flask Server](./custom/hello-world)
-
-[KFServing Custom Model](./custom/kfserving-custom-model)
-
-[Prebuilt Image](./custom/prebuilt-image)
-
-[BentoML](./bentoml)
-
-### Deploy KFServing InferenceService with Transformer
-[Image Transformer with PyTorch Predictor](./transformer/image_transformer)
-
-### Deploy KFServing InferenceService with Explainer
-[Alibi Image Explainer](./explanation/alibi/imagenet)
-
-[Alibi Text Explainer](./explanation/alibi/moviesentiment)
-
-[Alibi Tabular Explainer](./explanation/alibi/income)
-
-### Deploy KFServing InferenceService with Cloud or PVC storage
-
-[Models on S3](./s3)
-
-[Models on PVC](./pvc)
-
-[Models on Azure](./azure)
-
-### Deploy KFServing InferenceService with Autoscaling, Canary Rollout and Other Integrations
+## KFServing Features and Examples
+
+### Deploy InferenceService with Predictor
+KFServing provides a simple Kubernetes CRD to allow deploying trained models onto model servers such as [TFServing](https://www.tensorflow.org/tfx/guide/serving), 
+[ONNXRuntime](https://github.com/microsoft/onnxruntime), [Triton Inference Server](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs),
+[KFServer](https://github.com/kubeflow/kfserving/tree/master/python/kfserving). These model servers are also exposing a standardised API for both REST and gRPC. You could also choose to build your own model server for more complex use case,
+KFServing provides basic API primitives to allow you easily build custom model server, you can use other tools like [BentoML](https://docs.bentoml.org/en/latest) to build your custom model serve image.
+After models are deployed onto model servers with KFServing, you get all the following serverless features provided by KFServing
+- Scale to and from Zero
+- Request based Autoscaling on CPU/GPU
+- Revision Management
+- Optimized Container
+- Batching and Logger
+- Traffic management
+- Security with AuthN/AuthZ
+- Distributed Tracing
+- Out-of-the-box metrics
+- Ingress/Egress control
+
+|   | Exported model| HTTP | gRPC | Examples |
+| ------------- | ------------- | ------------- | ------------- | ------------- |
+| Deploy SKLearn Model on KFServer | pickled model(model.pkl, model.joblib) | :heavy_check_mark: | V2 |[SKLearn Iris](./sklearn)  |
```

Comment:
Can we move the exported model to the last column? Lot of the values are empty and breaks the readability flow